### PR TITLE
Try to add AlmaLinux 9

### LINKED ISSUE #388: Support RHEL9

### PR DESCRIPTION
**Pull request by ashie**
Fix #388

### CODE REVIEW

File: td-agent/yum/almalinux-9-aarch64/from~ [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (cosmo0920) [inline td-agent/yum/almalinux-9-aarch64/from~:1]: This file should be a backup file of editor? Not needed to included in this repo.

**Comment by cosmo0920** [comment]
We have to fix serverspec test cases for AlmaLinux 9.

**Comment by ashie** [comment]
> We have to fix serverspec preparing script for AlmaLinux 9.

Fixed. CI is passed with some more fixes. 
I'm now checking the built package.

**Comment by ashie** [comment]
Works fine on RHEL 9.

**Comment by ashie** [comment]
We are going to release the td-agent package for RHEL9 as td-agent 4.3.2 at the end of this month.

**Pull request merged**